array/fizz_buzz_array.rb

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+
+numbers_100 = (1..100).to_a
+fizz_buzz = ['Fizz', 'Buzz', 'FizzBuzz']
+numbers_100.each do |number|
```

Comment:
`.each` is an iterator method.  The assignment was to use an array method.  Perhaps `.map`?